mindocr/models/backbones/rec_abinet_backbone.py

```diff
@@ -0,0 +1,73 @@
+import mindspore.nn as nn
+
+from ...models.utils.abinet_utils import Model, PositionAttention, ResTranformer
+from ._registry import register_backbone, register_backbone_class
+
+__all__ = [
+    "ABINetIterBackbone",
+    "abinet_backbone"]
+
+
+# ABINet_backbone
+
+
+@register_backbone_class
+class ABINetIterBackbone(nn.Cell):
+    def __init__(self):
+        super().__init__()
+        self.iter_size = 3
+        self.max_length = 26
+        self.out_channels = 512
```

Comment:
Please don't define class members that are never used because it may be confusing for others.

Reply:
Modified, thank you.

Reply:
Mindocr requires class backbone to contain out_channels. Although this model is not used, it must be retained. The rest have been modified, thank you.